.github/workflows/build-sw-container.yml

```
@@ -9,19 +9,44 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  make-date-tag:
     runs-on: ubuntu-latest
     if: startsWith(github.repository, 'opensciencegrid/')
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
     steps:
-    - uses: actions/checkout@v2
-
     - name: make date tag
       id: mkdatetag
       run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+      
+  build:
+    runs-on: ubuntu-latest
+    needs: [make-date-tag]
+    if: startsWith(github.repository, 'opensciencegrid/')
+    strategy:
+      fail-fast: False
+      matrix:
+        repo: ['development', 'testing', 'release']
+    steps:
+    - uses: actions/checkout@v2
 
-    - id: format-docker-repo
-      run: echo "::set-output name=repo-name::${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}"
-
+    - id: generate-tag-list
+      env:
+        REPO: ${{ matrix.repo }}
+        TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
+      run: |
+        docker_repo=${GITHUB_REPOSITORY/mtru32\/docker-/mtru32/}
```

```suggestion
        docker_repo=opensciencegrid/cvmfs-nodeplugin
```

Based off the old tag list at the bottom of this doc